cpp/bin/torchtrtc/main.cpp

```diff
@@ -243,17 +52,23 @@ int main(int argc, char** argv) {
       "Require that the model should be fully compiled to TensorRT or throw an error",
       {"require-full-compilation"});
 
+  args::ValueFlag<std::string> check_method_op_support(
+      parser,
+      "check-method-op-support",
+      "Check the support for end to end compilation of a specified method in the TorchScript module",
+      {"supported", "is-supported", "check-support", "check-method-op-support"});
+
   args::Flag disable_tf32(
       parser, "disable-tf32", "Prevent Float32 layers from using the TF32 data format", {"disable-tf32"});
 
   args::Flag sparse_weights(
       parser, "sparse-weights", "Enable sparsity for weights of conv and FC layers", {"sparse-weights"});
 
-  args::ValueFlagList<std::string> enabled_precision(
+  args::ValueFlagList<std::string> enabled_precisions(
       parser,
       "precision",
       "(Repeatable) Enabling an operating precision for kernels to use when building the engine (Int8 requires a calibration-cache argument) [ float | float32 | f32 | fp32 | half | float16 | f16 | fp16 | int8 | i8 | char ] (default: float)",
-      {'p', "enabled-precision"});
+      {'p', "enable-precision"});
```

Comment:
So the reason why its not `enabled-precisions` is that enabled-precisions implies a set which is natural for the Python and C++ APIs because the fields are sets. But in the CLI uses repeated arguments not a set. So `enable-precision` is like a verb, like "enable fp16 precision".

Reply:
Seems to me to be more natural for someone who is just reading the man page to figure out how to use the CLI

Reply:
Okay cool